test/integration/ads_integration.h

```diff
@@ -37,13 +37,12 @@ class AdsDeltaSotwIntegrationSubStateParamTest
   Grpc::SotwOrDelta sotwOrDelta() const { return std::get<2>(GetParam()); }
 };
 
-class AdsIntegrationTest : public AdsDeltaSotwIntegrationSubStateParamTest,
-                           public HttpIntegrationTest {
+class AdsIntegrationTestBase : public Grpc::BaseGrpcClientIntegrationParamTest,
```

Comment:
do we expect multiple parallel subclasses of `AdsIntegrationTestBase` class in future? If not, could just keep is simple with one base class